code/datums/gamemode/factions/legacy_cult/cult_structures.dm

```diff
@@ -18,65 +18,6 @@
 	desc = "A forge used in crafting the unholy weapons used by the armies of Nar-Sie."
 	icon_state = "forge"
 
-/*
-/obj/structure/cult/pylon
```

Comment:
I'm not removing these, Gurfan had re-defined them in bloodcult_buildings.dm https://github.com/vgstation-coders/vgstation13/blob/Bleeding-Edge/code/datums/gamemode/factions/bloodcult/bloodcult_buildings.dm#L2071

So I removed the obsolete code from there after salvaging the useful damage procs and stuff (since currently simple mobs and unarmed humans cannot damage pylons at all)